Externals/fmt/include/fmt/format.h

```diff
@@ -2198,7 +2198,7 @@ FMT_CONSTEXPR20 auto do_write_float(OutputIt out, const DecimalFP& fp,
     -> OutputIt {
   auto significand = fp.significand;
   int significand_size = get_significand_size(fp);
-  static constexpr Char zero = static_cast<Char>('0');
+  const Char zero = static_cast<Char>('0');
```

Comment:
Shouldn't this be part of a separate PR updating the fmt external to at least `3e28dc021cf38b14ac0e2b1a046f783e094bfffc`?

Reply:
We should be using a git submodule but I was lazy.

Reply:
We have patches on fmt that would need to be taken care of upstream